tests/backend-tests.csproj

```diff
@@ -27,10 +27,18 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="itext7" Version="7.2.2" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\app\backend.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="itext7" />
```

Comment:
Co to :O

Reply:
Dzięki, że zwróciłeś na to uwagę. Znowu Visual nam dodał na siłę. 😸